opentelemetry-otlp/src/exporter/tonic.rs

```diff
@@ -25,13 +27,32 @@ pub struct TonicConfig {
 ///
 /// [tonic]: <https://github.com/hyperium/tonic>
 /// [channel]: tonic::transport::Channel
-#[derive(Default, Debug)]
+#[derive(Debug)]
 pub struct TonicExporterBuilder {
     pub(crate) exporter_config: ExportConfig,
     pub(crate) tonic_config: TonicConfig,
     pub(crate) channel: Option<tonic::transport::Channel>,
 }
 
+impl Default for TonicExporterBuilder {
+    fn default() -> Self {
+        let tonic_config = TonicConfig {
+            metadata: Some(MetadataMap::from_headers(
+                (&default_headers())
+                    .try_into()
+                    .expect("Invalid tonic headers"),
+            )),
+            ..Default::default()
```

Comment:
```suggestion
        let tonic_config = TonicConfig {
            metadata: Some(MetadataMap::from_headers(
                (&default_headers())
                    .try_into()
                    .expect("Invalid tonic headers"),
            )),
            #[cfg(feature = "tls")]
            tls_config: None,
```

Should do the trick

Reply:
makes sense! I tried to do something like this but obviously did it wrong, it wouldn't compile :) this compiles, so i'll run it through! Thanks!